testbot.py

```diff
@@ -74,6 +73,23 @@ async def handle_command(message: discord.message):
         return
 
 
+async def dadbot(message: discord.message):
+    content = message.content
+
+    if len(content) > 69 or content.count('.') > 2:
+        return
+
+    dadbot_pattern = re.compile('[Ii]\'?[Mm] (?P<mocked_name>\w+)')
+    m = re.match(dadbot_pattern, content)
```

Comment:
Doesn't seem to work if the pattern is not in the beginning of the sentence. Ideally, you can use an if statement to see if that pattern is in the message

Kinda like this, but instead of passing the string, maybe try to pass the pattern if that's possible
https://stackoverflow.com/questions/19300020/python-match-a-string-with-regex